app/controllers/exchanges/hbx_profiles_controller.rb

```diff
@@ -184,50 +183,48 @@ def employer_datatable
     # copy the link and open in new tab
     last_visited_url = current_user.try(:last_portal_visited) || root_path if current_user.present?
     @datatable = Effective::Datatables::BenefitSponsorsEmployerDatatable.new
-      respond_to do |format|
-       format.html { redirect_to(last_visited_url) }
-       format.js
-      end
+    respond_to do |format|
+      format.html { redirect_to(last_visited_url) }
+      format.js
+    end
   end
 
-def employer_poc
-
-    # Dynamic Filter values for upcoming 30, 60, 90 days renewals
+  def employer_poc
+      # Dynamic Filter values for upcoming 30, 60, 90 days renewals
     @next_30_day = TimeKeeper.date_of_record.next_month.beginning_of_month
     @next_60_day = @next_30_day.next_month
     @next_90_day = @next_60_day.next_month
 
     @datatable = Effective::Datatables::EmployerDatatable.new
     render '/exchanges/hbx_profiles/employer_poc'
- #   respond_to do |format|
-  #    format.html
-   #   format.js
-   # end
+   #   respond_to do |format|
+    #    format.html
+     #   format.js
+     # end
   end
 
-
   def staff_index
     @q = params.permit(:q)[:q]
     @staff = Person.where(:$or => [{csr_role: {:$exists => true}}, {assister_role: {:$exists => true}}])
     @page_alphabets = page_alphabets(@staff, "last_name")
     page_no = cur_page_no(@page_alphabets.first)
-    if @q.nil?
-      @staff = @staff.where(last_name: /^#{Regexp.escape(page_no)}/i)
-    else
-      @staff = @staff.where(last_name: @q)
-    end
+    @staff = if @q.nil?
+               page_no.present? ? @staff.where(last_name: /^#{Regexp.escape(page_no)}/i) : []
```

Comment:
The previous version had a little different flow. Just for sure.
Are we sure it works as we expect?